### ISSUE TITLE
Pre-trained models

### ISSUE THREAD

**Issue by pwwang** [jasminezhoulab/cfSort#8]
Hi,

I found this description about the pretrained models in the README:

> We provide two pre-trained DNNs which are the component DNNs of the cfSort: DNN1.h5 and DNN2.h5. Due to the limit of file size on GitHub, we deposit these two models on https://zenodo.org/record/7884243. cfSort ensembles the two DNNs by averaging the tissue compositions.

From the description, I can confirm that the final results are the average output of these two models. However, I am still not clear about:

1. What is the difference between these two models?
2. If we are planning to train our own models, how can we train these two models?
3. Is it possible to train a single model, if we are training our own model from scratch?

Thank you so much in advance!

**Comment by DashuoLI** [comment]
Hi,

1. The two models have distinct network structures, for example, number of nodes and dropout rates. The output from the two models can be slightly different between the two models. Please see Fig3b in our manuscript for the details. 
2. You may start from tissue samples. For our models, we generated in silico cfDNA samples  by mixing tissue samples at different tissue compositions. We described our simulation strategy in our manuscript and showed that the DNNs trained from our simulation data can be applicable to cfDNA data. Due to the limit of the data size (>295K samples), we didn't put the simulation data on the GitHub. 
3. It may be possible to train a single model. In this work, we hoped to take advantage of different DNN structures, which may learn different tissue representations in their hidden layers. Therefore, we chose to use an ensemble model.

Best,
Shuo

**Comment by pwwang** [comment]
Thanks for the prompt reply! 

So is the code shared here (model_train.py) for the first architecture (DNN1) or the second (DNN2)?

**Comment by DashuoLI** [comment]
The two DNNs both used model_train.py, but they used different input architecture parameters (e.g. line 13 and 14).

**Comment by pwwang** [comment]
That helps a ton! Thank you!
